R/addRank.R

```diff
@@ -0,0 +1,17 @@
+setGeneric("addRank", signature = c("x"),
+           function(x,
+                    method = c("average", "first", "last", "random", "max", "min"))
+             standardGeneric("addRank"))
+
+setMethod("addRank", signature = c(x = "SummarizedExperiment"),
+          function(x,
+                   method = c("average", "first", "last", "random", "max", "min")){
+          col_data <- colData(x)
+          if (!is.vector(col_data)){
+            col_data <- as.vector(col_data)
```

Comment:
indentation..? Also run the following ones and make sure there are no errors.

```
document()
run_examples()
check()
BiocCheck::BiocCheck()
```

Then we are already pretty far.